cmd/kubeadm/app/phases/copycerts/copycerts_test.go

```diff
@@ -25,19 +25,23 @@ import (
 	"testing"
 
 	"github.com/lithammer/dedent"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	fakeclient "k8s.io/client-go/kubernetes/fake"
+	certutil "k8s.io/client-go/util/cert"
+	keyutil "k8s.io/client-go/util/keyutil"
 	kubeadmapi "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
 	kubeadmconstants "k8s.io/kubernetes/cmd/kubeadm/app/constants"
+	"k8s.io/kubernetes/cmd/kubeadm/app/phases/certs"
 	cryptoutil "k8s.io/kubernetes/cmd/kubeadm/app/util/crypto"
 	testutil "k8s.io/kubernetes/cmd/kubeadm/test"
 )
 
-func TestUploadCerts(t *testing.T) {
-	tmpdir := testutil.SetupTempDir(t)
-	defer os.RemoveAll(tmpdir)
-
-}
+const (
+	keyFileMode  = 0600
+	certFileMode = 0644
```

Comment:
these constants a bit tricky to handle because the whole package can see them.

if multiple functions/files use them we should move to new file constants.go.
but it seems like only TestUploadCerts uses them, in which case we should just move them to that function for now.